utils/change_engine/ordinary_to_atomic.py

```diff
@@ -1,83 +1,75 @@
 #!/bin/python3
 from clickhouse_driver import Client
+from interface import Database
 import sys
 
 
+class UserInteractor:
+    def ask(self, text):
+        answer = input(text)
+        return answer
+
+    def notify(self, text): #TODO logging
+        print(text)
+
+
 class DBOrdinaryToAtomicConverter:
     def __init__(self):
         self.__atomic_prefix = '__temp_ordinary_to_atomic__'
         self.__client = Client('localhost')
-
-    def __db_exist(self, db_name):
-        dbs = self.__client.execute(f'SELECT * FROM system.databases WHERE name=\'{db_name}\'')
-        return len(dbs) == 1
-
-    def __db_engine(self, db_name):
-        assert self.__db_exist(db_name), 'something went wrong'
-
-        dbs = self.__client.execute(f'SELECT engine FROM system.databases WHERE name=\'{db_name}\'')
-        return dbs[0][0]
-
-    def __get_tables(self, db_name):
-        tables = self.__client.execute(f'SELECT name FROM system.tables WHERE database == \'{db_name}\'')
-        return tables
-
-    def __continue_change(self, atomic_name):
-        self.__rename_atomic_db(atomic_name)
-
-    def __abort_change(self, atomic_name):
-        self.__drop_database(atomic_name)
-
-    def __create_atomic_db(self, ordinary_name):
-        atomic_name = f'{self.__atomic_prefix}{ordinary_name}'
-
-        if self.__db_exist(atomic_name):
-            if self.__db_exist(self.__remove_atomic_prefix(atomic_name)):
-                self.__abort_change(atomic_name)
-                print('The last launch failed but ordinary table was restored')
+        self.__user_interactor = UserInteractor()
+
+    def __get_atomic_name(self, ordinary_name):
+        return f'{self.__atomic_prefix}{ordinary_name}'
+
+    def __finished_previous_sessions(self, ordinary_database, atomic_database):
```

Comment:
It is rather `temp_database` or something like that than `atomic_database`

Reply:
This function's name doesn't tell what does it do. With this name it should return either to continue, revert or cancel.